Cargo.toml

```diff
@@ -51,14 +50,12 @@ walkdir = "2.5.0"
 sqd-contract-client = { git = "https://github.com/subsquid/sqd-network.git", version = "1.0.7" }
 sqd-messages = { git = "https://github.com/subsquid/sqd-network.git", version = "1.1.4" }
 sqd-network-transport = { git = "https://github.com/subsquid/sqd-network.git", version = "1.0.25", features = ["worker", "metrics"] }
+sqd-query = { git = "ssh://git@github.com/subsquid/data.git", version = "0.1.0", features = ["parquet"] }
+sqd-polars = { git = "ssh://git@github.com/subsquid/data.git", version = "0.1.0" }
```

Comment:
Let's fix the revision here explicitly, but there will be `sqd-query` update before the release.